.github/styles/Vocab/FluxNinja/accept.txt

```diff
@@ -50,6 +50,23 @@ Prometheus
 Pub/Sub
 [Aa]uto-scalers?\b
 [Rr]ollouts?\b
+LinkedIn
+Nutanix
+Netsil
+Kubernauts
+SLAs
+vs
```

Comment:
We shouldn't use vs very often.

Reply:
Should we remove it from accepted? otherwise folks will keep using it

Reply:
Sure

Reply:
Be very selective with words that we add here